csharp/ql/src/semmle/code/csharp/dataflow/DataFlow.qll

```diff
@@ -1087,7 +1099,7 @@ module DataFlow {
       predicate localFlowExit(Node node, Configuration config) {
         Pruning::nodeCand(node, config) and
         (
-          jumpStep(node, _) or
+          additionalJumpStep(node, _, config) or
```

Comment:
I think we should run a dist comparision to check that performance does not regress with these changes.